examples/lb3-application/lb3app/common/models/coffee-shop.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const debug = require('debug')('loopback:example:lb3application');
+
+module.exports = function(CoffeeShop) {
+  CoffeeShop.status = function(cb) {
+    var currentDate = new Date();
+    var currentHour = currentDate.getHours();
+    var OPEN_HOUR = 6;
+    var CLOSE_HOUR = 20;
```

Comment:
Let's use `const` instead of `var` please, see https://loopback.io/doc/en/contrib/style-guide-es6.html#variable-declarations

This comment applies to all newly added JS code.

In the future, I'd like us to enable linter rule to automatically detect this old style, but we are not there yet - see #2492 and #2572.